test/stubby_test.exs

```diff
@@ -16,6 +16,16 @@ defmodule StubbyTest do
     use Stubby, for: [StubbyTest.FakeBehaviour, StubbyTest.FakeBehaviour2]
   end
 
+  defmodule TestModule do
+    def some_function, do: :ok
+    def some_function(arg), do: {:ok, arg}
+    def some_other_function, do: :ok
+  end
+
+  defmodule TestModuleStub do
+    use Stubby, module: StubbyTest.TestModule
```

Comment:
💯

Reply:
How about some tests to verify that functions can be overridden using this syntax.  It seems pretty straightforward but probably also worth documenting via tests.

Reply:
Also, probably an issue for future consideration but what do you think about additional params for `only: []`, and `except:[]` which could allow for some of the original function(s) to be carried over to the stubbed module.  I'm not sure how practical this idea is but it seems like similar features have come in handy in other languages / mocking frameworks.

Reply:
Thanks Joel! I dig these. I added a few issues to add support for this. I thought your suggestions were great (hence the issue creation). Thanks for the feedback! 👍